src/components/checkbox/__tests__/Checkbox.stories.test.js

```diff
@@ -0,0 +1,12 @@
+describe('components/checkbox/Checkbox', () => {
+    test.each([
+        ['components-checkbox--basic'],
+        ['components-checkbox--controlled'],
+        ['components-checkbox--disabled'],
+        ['components-checkbox--withTooltip'],
```

Comment:
A few of the `-snap.png` files are showing error messages: "sorry but you either have no stories or none are selected somehow". The names here might need to be `with-tooltip`, `with-subsection`. See `components-checkbox-with-subsection-1-snap.png`, `components-checkbox-with-tooltip-1-snap.png`